READMEを英語に翻訳 (#76)
standfirm/thinreports-generator: README_FOR_SECTION_REPORT.md

# SectionReport

A section-report format is a new template file format that is proposed at [the feature concept by Thinreports community](https://github.com/thinreports/thinreports/issues/7).

## Getting Started

### Prerequisites

To generate PDF with the section-report format, you need a template (tlf) file for the section-report format.
You have to use [this new Thinreports Editor](https://github.com/misoca/thinreports-editor) to create a template file for the section-report format.

> ### Installation

Reviewer: https://gist.github.com/PurpleBooth/109311bb0361f32d87a2 を参考にしているので、Installing にしておきたい。また、下の Generating PDF とも統一できるので。

> Add the line below to your Gemfile.

Reviewer: below はなくても良いかも。どっちでも良い

```suggestion
Add this line to your Gemfile:
```

- [StackView with Floating Item](test/features/section_report_stack_view_with_floating_item/README.md)
- [TextBlock Vertical Align](test/features/section_report_text_block_vertical_align/README.md)

> ## Current Implementation Status

Reviewer: 'Status' が 'Current' の意味を含むのは自明なので

```suggestion
## Implementation Status
```

または、単に `Status` とか `Feature Status` とかでも良いかもしれない

Here is the current implementation status based on [the feature concept by Thinreports community](https://github.com/thinreports/thinreports/issues/7)

### Not Implemented

- `fixed at the bottom` property for footer sections
- page-number item

### Additional Features

- stack-view item
  - See [StackView](test/features/section_report_stack_view/README.md) for details
- `offset_x` and `offset_y` parameters for item position translation
  - See [Item Parameter](test/features/section_report_item_parameters/README.md) for details

>   - Limitation: Currently only image-blocks are supported

Reviewer: Currently はなくても良いかも

Reviewer: なんとなく違和感があるんですが、

```suggestion
  - Limitation: only supports image-block item
```

か、 `Limitation: available for image-block only' とかかなぁ。このままでも問題はないと思います (正直わからん)

> - automatic stretch feature that supports not only extension but also shrink

Reviewer: ここ難しいですよね。日本語の文章が悪い。「自動縮小のサポート」だけで良いかもしれない。

Reviewer: `Additional Features` というセクションだから、「〜のサポート」ではなく、単に「自動縮小」が正しいか

  - See [Section Auto Stretch](test/features/section_report_section_auto_stretch/README.md) and [StackViewRow Auto Stretch](test/features/section_report_stack_view_row_auto_stretch/README.md) for details

>   - Limitation: The shrink of a section or a stack-view by image-blocks works only when its `vertical-align` property is `top`. That shrink occurs when the height of a drawn image is lower than the height of the image-block.

Reviewer: ここも日本語文章が悪いですね... image-block による縮小の仕様については README で説明しているので、ここでは単に

「image-block による section 又は row の高さの縮小は image-block が上揃えのときのみサポート」

という内容で良いと思います。よって、

Section and stack-view-row shrink only by image-block that has vertical-align property is top.

とか。これだと縮小そのものが image-block のみでしか起きないという意味になるか。

Author: 以下のような感じでいいですかね

> Sections and stack-view-rows shrink only by image-block items whose `vertical-align` property is `top`.

Reviewer: LGTM

> - `follow-stretch` property, by which item layout changes automatically when the height of the section or the stack-view-row changes

Reviewer: ```suggestion
- `follow-stretch` property, which is a property of whether to change the layout of the item according to the stretch of the section or stack-view-row
```

もしくは、README で詳細は説明しているので、単に `follow-stretch` property だけでも良いかも

Author: 以下のような感じでいいですかね

> `follow-stretch` property to change the layout of an item according to the stretch of the section or stack-view-row

Reviewer: 良いと思いますが、追加した機能 (`follow-stretch` プロパティ) と、その説明 (which ~) をわけるとわかりやすいかなと思って suggestion のようにしています。ただ、英語的に正しいのかはわからない。

Author: なるほど、このような感じでどうでしょう？

> `follow-stretch` property, which specifies how to change the layout of an item according to the stretch of the section or stack-view-row

Reviewer: 良いですね

  - See [Item Follow Stretch](test/features/section_report_item_follow_stretch/README.md) for details

>   - Limitation: Currently some types of items are not supported

Reviewer: currently は消しつつ、
```suggestion
  - Limitation: Some item types are not supported
```

か、`Limitation: Only some item types support` とかかな

### Known Issues

- performance problem

>   - The height calculations of sections or stack-view-rows are called several times for a single PDF rendering

Reviewer: `for a single PDF rendering` は自明なのでなくても良いと思う

  - See [StackView](test/features/section_report_stack_view/README.md) for details
- `offset_x` and `offset_y` parameters for item position translation
  - See [Item Parameter](test/features/section_report_item_parameters/README.md) for details

>   - Limitation: only supports image-block items

Reviewer: ```suggestion
  - Limitation: Only supports image-block items
```

他の箇所は大文字から開始しているので

Reviewer: コミットを入れておきます
